gravitee-am-ui/src/app/domain/settings/password-policy/domain-password-policy.component.ts

```diff
@@ -140,6 +145,31 @@ export class DomainPasswordPolicyComponent implements OnInit {
     this.passwordPolicy = { ...policy };
   }
 
+  private buildLinkedIdentityProviders(areProvidersLoaded: boolean) {
+    if (areProvidersLoaded) {
+      // rebuild the list of selected IDP based on the modal result
+      // add new idp into the list
+      this.providers
+        .filter((prov) => this.idpToUpdate.get(prov.id))
+        .map((prov) => this.constructDataModel(prov))
+        .forEach((prov) => this.linkedProviders.push(prov));
```

Comment:
it can be used concat function instead forEach (..push)

this.linkedProviders.concat(...